spec/acceptance/github/pull_request_opened_spec.rb

```diff
@@ -15,8 +15,15 @@
         "/issue/jt-1234/comment",
         expected_body
       )
+      expected_body_2 = "{\"fields\":{\"Branch Name\":\"jt-1234-create-crawler\"}}"
```

Comment:
Make a dedicated `it` test, e.g. `it "sets the branch name field with the branch name"`